dallinger/recruiters.py

```diff
@@ -447,6 +658,40 @@ def close_recruitment(self):
         # except MTurkServiceException as ex:
         #     logger.exception(str(ex))
 
+    def _mturk_status_for(self, participant):
+        try:
+            assignment = self.mturkservice.get_assignment(participant.assignment_id)
+            status = assignment['status']
+        except Exception:
+            status = None
+        return status
+
+    def _disable_autorecruit(self):
```

Comment:
I know you just moved this, rather than writing it, but do you think we should handle this a bit more generically, rather than assuming we're on Heroku as we're running with mturk?